action.yml

```diff
@@ -2,6 +2,10 @@ name: 'CI Standard Checks'
 description: 'Runs Standard Checks for Typeform'
 author: 'Typeform'
 inputs:
+  skipChecks:
+    required: true
```

Comment:
It has a default of `''`. This makes it easier to guarantee that it will have a value that I'm handling correctly in the implementation (I have a test case for `''`) instead of relying on it being `null` or `undefined`

Reply:
You can also see that in out test-and-release workflow where we run all checks without providing the `skipChecks` input. It behaves same as before